docs/databags.md

```diff
@@ -93,7 +93,10 @@ An apps hash is a contextual (see above) Hash, part of a plaintext data bag item
 * `['bag']` - A string that points to an externalized Apps Hash in which all keys (except this one) are valid. 
 * `[app]` - The name of an app to manage (disk name)
```

Comment:
Thanks.  I get it all now.

Maybe worth a slight amendment to [the description of the 'url' attribute](https://github.com/cerner/cerner_splunk/pull/48/files#diff-a8cd9604d38ad43b8b92ae521a075a6aR98) to say something like the following?

"URL of where to download the app .tar.gz or .spl file.  Archive is expected to contain top-level directory with name matching 'app' attribute above."